### ISSUE TITLE
Output for new predictions

### ISSUE THREAD

**Issue by nschneid** [xiaoling/figer#1]
Thanks for the useful tool! When running on new data, is there a way to write the predictions in BIO format to a file?

**Comment by xiaoling** [comment]
Hey,

Sorry for the late reply. Gmail misclassified the notification email into a wrong category, which makes me miss it.

Yes, you can get a BIO-formatted output by running
$ ./run.sh "config/figer.conf"
(You will need to set the input/output file at `config/figer.conf`).

Please see my recent commit (c5f7722) for more details and let me know if you have more questions.

Thanks,
Xiao

**Comment by nschneid** [comment]
This is giving me a compile error:

```
[info] Compiling 1 Java source to figer/target/scala-2.10/classes...
[error] figer/src/main/java/edu/washington/cs/figer/FigerSystem.java:237: ')' expected
[error] figer/src/main/java/edu/washington/cs/figer/FigerSystem.java:248: not a statement
[error] System.out.println("[l"+i+"][s" + sentId + "]mention" + mention
[error] 							+ "(" + offset.first +","+offset.second)+" = "+mention
[error] 							+ ", pred = "
[error] figer/src/main/java/edu/washington/cs/figer/FigerSystem.java:248: ';' expected
```
```
$ javac -version
javac 1.7.0_79
```

**Comment by xiaoling** [comment]
sorry that was stupid. now it's fixed.

**Comment by nschneid** [comment]
OK, it's working for the example. When I substitute an example with multiple sentences, though, all the preprocessing happens correctly, but only the final sentence appears in the .out file.

**Comment by xiaoling** [comment]
hmm.. Can you share the file? My run with multiple sentences predicts for all the sentences.

**Comment by nschneid** [comment]
All on one line:

    Sounds haunting , and a little bit Harry Potter , but I want to check it out .  YEAR 3 : Hermione held her own when Draco Malfoy and his minions gave her grief .  The silver screen adventures of Harry , Ron and Hermione have been a magic pot of gold for Hollywood studio Warner Bros , with the seven films released so far grossing $ 6.4 billion in ticket sales and billions more from DVDs and merchandise .

**Comment by xiaoling** [comment]
Thanks for the feedback. I made a fix (906ed90). Please give it another try and let me know if there are more issues.

**Comment by nschneid** [comment]
Perfect, thanks!
